src/main/java/com/checkmarx/flow/controller/GitHubController.java

```diff
@@ -280,7 +299,7 @@ public String pingRequest(
             Repository repository = event.getRepository();
             String gitUrl = repository.getCloneUrl();
             log.debug("Using url: {}", gitUrl);
-            String token = scmConfigOverrider.determineConfigToken(properties, controllerRequest.getScmInstance());
+            String token = gitHubAuthService.getBasicAuth(controllerRequest.getScmInstance());
```

Comment:
this will disable the config override, we need to see how to support both logics